### PR TITLE
Install add 'kubectl cert-manager x install' documentation

### PR DESCRIPTION
**Pull request by inteon**

### CODE REVIEW

File: content/en/docs/installation/kubectl-plugin.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+The plugin provides the simplest way of installing cert-manager:
+```bash
+$ kubectl cert-manager x install
+```
+The command makes sure that the required `CustomResourceDefinitions` are installed together with the cert-manager, cainjector and webhook components.
+Under the hood, a procedure similar to the [Helm install procedure](../helm/#steps) is used.
+
+All Helm templating parameters can also be used with the kubectl plugin.
+On the [cert-manager's ArtifactHub page](https://artifacthub.io/packages/helm/cert-manager/cert-manager) a full list of the available **Helm values** can be found.
```
Comment (wallrj) [inline content/en/docs/installation/kubectl-plugin.md:27]:
```suggestion
You can also use `kubectl cert-manager x install` to customize the installation of cert-manager.
```

File: content/en/docs/installation/kubectl-plugin.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The example below shows how to tune the cert-manager installation by overwriting the default Helm values:
+
+```bash
+$ kubectl cert-manager x install \
+    --set prometheus.enabled=false \  # Example: disabling prometheus using a Helm parameter
+    --set webhook.timeoutSeconds=4s   # Example: changing the wehbook timeout using a Helm parameter
+```
+
```
Comment (wallrj) [inline content/en/docs/installation/kubectl-plugin.md:36]:
```suggestion
You can find [a full list of the install parameters on cert-manager's ArtifactHub page](https://artifacthub.io/packages/helm/cert-manager/cert-manager#configuration). These are the same parameters that are available when using the Helm chart.
```

File: content/en/docs/installation/kubectl.md [hunk 3]
```diff
@@ -29,8 +29,7 @@ By default, cert-manager will be installed into the `cert-manager`
 namespace. It is possible to run cert-manager in a different namespace, although
 you'll need to make modifications to the deployment manifests.
 
-Once you have deployed cert-manager, you can verify the installation
-[here](../verify/).
+Once you have deployed cert-manager, you can [verify](../verify/) the installation.
```
Comment (wallrj) [inline content/en/docs/installation/kubectl.md:32]:
```suggestion
Once you have deployed cert-manager, you can [verify the installation](../verify/) .
```

File: content/en/docs/installation/kubernetes.md [hunk 4]
```diff
@@ -1,342 +0,0 @@
----
```
Comment (wallrj) [inline content/en/docs/installation/kubernetes.md:1]: Should these have been deleted in #647  ?

**Review by wallrj — APPROVED** [review]
Looks good @inteon 

I couldn't help suggesting some changes, but I'm not sure if I've made the words clearer or more tortured. 
Take or leave the suggestions.

Shame we can't see a preview of this :frowning_face:

File: content/en/docs/installation/kubernetes.md (hunk 4 above)
Comment (inteon) [reply content/en/docs/installation/kubernetes.md:1]: Yes, and they were; I asked @maelvls to rebase and that might have added them again.
Not sure why there was a merge conflict...

Comment (maelvls) [reply content/en/docs/installation/kubernetes.md:1]: Arghh, I'll look into it. That might be due to the merge I did ( ab959b5d085ee05f91f42dc509b138d3a084dd69).

Comment (maelvls) [reply content/en/docs/installation/kubernetes.md:1]: I mistakenly resolved the conflict between ada4f5ab02171765bf493037741965d1ed6c78a1 and 3186145ecaf410b6536f77115286cc507fc65a01 by choosing 3186145ecaf410b6536f77115286cc507fc65a01... my fault

Really anoying that Git does not record any conflict resolution information. Things like `git log -p --cc ab959b5` are useless when looking for why a file has disappeared/appeared after a conflict...

**Review by wallrj — CHANGES_REQUESTED** [review]
One final request: please add a short note to the release-notes-v1.5 page describing the new installer. Thanks.

https://github.com/cert-manager/website/blob/release-next/content/en/docs/release-notes/release-notes-1.5.md

**Review by wallrj — APPROVED** [review]
Great.

/lgtm

**Pull request merged**